docs/examples/basic_usage/memory_benchmark.py

```diff
@@ -0,0 +1,169 @@
+import json
+from argparse import ArgumentParser
+from contextlib import nullcontext
+from os import path
+
+from benchmark_utils import GPTWrapper
+from example_benchmark import (
+    HAS_JVP,
+    LINOP_STRS,
+    OP_STRS,
+    PROBLEM_STRS,
+    SKIP_EXISTING,
+    benchpath,
+    setup_linop,
+    setup_problem,
+)
+from memory_profiler import memory_usage
+from torch import cuda, device, manual_seed, rand
+from torch.nn.attention import SDPBackend, sdpa_kernel
+
+from curvlinops import KFACInverseLinearOperator, KFACLinearOperator
+
+
+def run_peakmem_benchmark(  # noqa: C901
+    linop_str: str, problem_str: str, device_str: str, op_str: str
+):
+    """Execute the memory benchmark for a given linear operator class and save results.
+
+    Args:
+        linop_str: The linear operator.
+        problem_str: The problem.
+        device_str: The device.
+        write_results: Whether to write the results to a file. Default is ``True``.
```

Comment:
This argument doesn't exist and there is not entry for `op_str`.

Reply:
Thanks